examples/server/public/index.html

```diff
@@ -536,26 +620,26 @@
       }
 
       return html`
-        <form onsubmit=${submit}>
-          <div>
-            <textarea
-               className=${generating.value ? "loading" : null}
-               oninput=${(e) => message.value = e.target.value}
-               onkeypress=${enterSubmits}
-               placeholder="Say something..."
-               rows=2
-               type="text"
-               value="${message}"
-            />
-          </div>
-          <div class="right">
-            <button type="submit" disabled=${generating.value}>Send</button>
-            <button onclick=${uploadImage}>Upload Image</button>
-            <button onclick=${stop} disabled=${!generating.value}>Stop</button>
-            <button onclick=${reset}>Reset</button>
-          </div>
-        </form>
-      `
```

Comment:
This would be easier to review if there weren't these extra whitespace changes?

Reply:
Synced some of spacing issue, I think editorconfig didn't work as expected

Reply:
There's still lots of extra whitespace changes.

Reply:
updated some more and added new screenshots.